rest/authorization-server/pom.xml

```diff
@@ -122,7 +123,7 @@
         <dependency>
             <groupId>org.springframework.restdocs</groupId>
             <artifactId>spring-restdocs-core</artifactId>
-            <version>${spring-restdocs.version}</version>
+            <version>3.0.1</version>
```

Comment:
Please use the property for the version.

Reply:
Done